src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -99,13 +90,20 @@ const GiftPopover = (
   </Popover>
 );
 
-const navBarScheduleClassName = "NavBar__schedule-dropdown";
+const VolumePopover = (
+  <Popover
+    id="volume-popover"
+    className="NavBar__volume-popover NavBar__volume-popover--without-sliders"
+  >
+    <Popover.Content>
+      <VolumePopOverContent />
+    </Popover.Content>
+  </Popover>
+);
```

Comment:
This is basically its own component. If properly isolated, one can even deal with the `id` being prefixed random string.

[OOS]
On that note. For the long term, but probably done as we go, anything that can be extracted out of the `NavBar.tsx` and `NavBar.scss` as a separate component/hook/utility should probably be done in order to improve code maintenance